clj-refactor.el

```diff
@@ -379,9 +379,15 @@ if SAVE-EXCURSION is T POINT does not move."
           (search-forward s bound t))
       (search-forward s bound t))))
 
+(defun cljr--looking-back (regexp &optional limit greedy)
+  "Call `looking-back', but tolerate limit > point."
```

Comment:
You're right.  In fact all calls to look-back specifying limit in this file is using it wrong.  

In fact using limit isn't really necessary if we use the \= boundary.   

I'll close this and make another PR.

Thanks @expez